cp2/ControlPanel/ControlPanel/view.py

```diff
@@ -152,6 +153,22 @@ def ajax_add_demo(request):
     settings = {"state": state, "title": title, "demo_id": demo_id, "ddl": "{}"}
     result, demo_status = api_post("/api/demoinfo/demo", method="post", params=settings)
 
+    response = {}
+    if demo_status != 201:
+        # response = JsonResponse({"status": "OK"})
+        # response["HX-Trigger"] = json.dumps(
+        #     {
+        #         "show-toast": {
+        #             "level": "error",
+        #             "title": "Error",
+        #             "message": "This demo ID is already taken",
+        #         }
+        #     }
+        # )
+        # return response
```

Comment:
```suggestion
```

Reply:
Sorry, I really thought I had removed those comments before adding to git